custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -49,7 +58,18 @@ def aggregation_query(self):
                    awcm.block_name, awcm.supervisor_name, awcm.awc_name, awcm.aww_name,
                    awcm.contact_phone_number, awcm.wer_weighed, awcm.wer_eligible,
                    awcm.awc_days_open
-        )
+        );
+        /* update expected visits for cf cases (not in agg_ccs_record */
+        UPDATE {tablename} perf
+        SET expected_visits = expected_visits + ucr.expected
+        FROM (
+             SELECT SUM(0.39) AS expected, awc_id
+             FROM {ccs_record_case_ucr} 
+             WHERE %(month)s - add > 183 AND closed_on IS NULL OR date_trunc('month', closed_on)::DATE >= %(month)s 
```

Comment:
I think that is expected, in so far as it is the current behavior for other cases, because they are excluded from the calculation in `agg_ccs_record`. I don't think calculating based on what was open from the beginning fo the month makes any more sense because it has the same concern but in reverse. In order to err on the side of lower expected visits, which has been the general rule, we kept this. In theory we could adjust the count by what percent of the month the case was open, but that doesnt really translate to how the home visit scheduler works.

Reply:
> in so far as it is the current behavior for other cases

do you mean that as a general rule for the dashboard or this report? the general rule for the dashboard would be closed on < next_month or close_on <= last day of month. But this seems fine given your other explanation here